lib/Backend/BailOut.cpp

```diff
@@ -2161,6 +2181,21 @@ void BailOutRecord::ScheduleFunctionCodeGen(Js::ScriptFunction * function, Js::S
         rejitReason = RejitReason::Forced;
     }
 
+    // REVIEW: Temporary fix for RS1.  Disable Rejiting if it looks like it is not fixing the problem.
+    //         For RS2, turn this into an assert and let's fix all these issues.
+    if (!reThunk && rejitReason != RejitReason::None)
```

Comment:
Nit: Do you want to take care of -force:rejit scenario? Feel free to ignore the comment.